server/webapp/WEB-INF/rails/spec/helpers/application_helper_spec.rb

```diff
@@ -647,6 +647,38 @@ def is_user_an_admin?; true; end
     end
   end
 
+  describe "site_footer" do
+    it 'should return maintenance approver when server is in maintenance mode' do
+      expect(maintenance_mode_service).to receive(:isMaintenanceMode).and_return(true)
+      expect(maintenance_mode_service).to receive(:updatedOn).and_return("date")
+
+      expected = "date"
+      expect(maintenance_mode_updated_on).to eq(expected)
+    end
+
+    it 'should not return maintenance approver when server is in maintenance mode' do
+      expect(maintenance_mode_service).to receive(:isMaintenanceMode).and_return(false)
+
+      expected = nil
+      expect(maintenance_mode_updated_on).to eq(expected)
+    end
+
+    it 'should return maintenance mode update time when server is in maintenance mode' do
+      expect(maintenance_mode_service).to receive(:isMaintenanceMode).and_return(true)
+      expect(maintenance_mode_service).to receive(:updatedBy).and_return("bob")
+
+      expected = "bob"
+      expect(maintenance_mode_updated_by).to eq(expected)
```

Comment:
The test says we are checking for `update time` but the verification is on `updated by`.